doc/guides/element_value_types.rst

```diff
@@ -0,0 +1,127 @@
+| DA | Date             | :class:`str`    | :class:`str` or                   |
+|    |                  |                 | :class:`~valuerep.DA`\ :sup:`1`   |
++----+------------------+-----------------+-----------------------------------+
+| DS | Decimal String   | :class:`str`,   | :class:`~valuerep.DSfloat` or     |
+|    |                  | :class:`float`  | :class:`~valuerep.DSdecimal`\     |
+|    |                  | or :class:`int` | :sup:`2`                          |
++----+------------------+-----------------+-----------------------------------+
+| DT | Date Time        | :class:`str`    | :class:`str` or                   |
+|    |                  |                 | :class:`~valuerep.DT`\ :sup:`1`   |
++----+------------------+-----------------+-----------------------------------+
+| FL | Floating Point   | :class:`float`  | :class:`float`                    |
+|    | Single           |                 |                                   |
++----+------------------+-----------------+-----------------------------------+
+| FD | Floating Point   | :class:`float`  | :class:`float`                    |
+|    | Double           |                 |                                   |
++----+------------------+-----------------+-----------------------------------+
+| IS | Integer String   | :class:`str`    | :class:`~valuerep.IS`             |
+|    |                  | or :class:`int` |                                   |
++----+------------------+-----------------+-----------------------------------+
+| LO | Long String      | :class:`str`    | :class:`str`                      |
++----+------------------+-----------------+-----------------------------------+
+| LT | Long Text        | :class:`str`    | :class:`str`                      |
++----+------------------+-----------------+-----------------------------------+
+| OB | Other Byte       | :class:`bytes`  | :class:`bytes`                    |
++----+------------------+-----------------+-----------------------------------+
+| OD | Other Double     | :class:`bytes`  | :class:`bytes`                    |
++----+------------------+-----------------+-----------------------------------+
+| OF | Other Float      | :class:`bytes`  | :class:`bytes`                    |
++----+------------------+-----------------+-----------------------------------+
+| OL | Other Long       | :class:`bytes`  | :class:`bytes`                    |
++----+------------------+-----------------+-----------------------------------+
+| OV | Other 64-bit     | :class:`bytes`  | :class:`bytes`                    |
+|    | Very Long        |                 |                                   |
++----+------------------+-----------------+-----------------------------------+
+| OW | Other Word       | :class:`bytes`  | :class:`bytes`                    |
++----+------------------+-----------------+-----------------------------------+
+| PN | Person Name      | :class:`str`    | :class:`str` (Python 2) or        |
```

Comment:
in Python 2, there is also `PersonNameUnicode`

Reply:
I should add `unicode` for Python 2. From looking at [DataElement](https://github.com/pydicom/pydicom/blob/master/pydicom/dataelem.py#L540) I don't think either `str` or `unicode` values get stored as `PersonNameUnicode`, it only seems to happen when PN is read from file.

Might be a bug?

Reply:
This is current behavior as expected - this has been discussed earlier, and will not be changed for Python 2.

Reply:
Fair enough